bigquery/samples/utils.py

```diff
@@ -21,9 +21,16 @@ def get_service():
     from oauth2client.client import GoogleCredentials
 
     credentials = GoogleCredentials.get_application_default()
+
```

Comment:
iirc, this is handled automatically by the library. https://github.com/google/google-api-python-client/blob/master/googleapiclient/discovery.py#L272